### DISCUSSION TITLE
Name mapping on post

### DISCUSSION THREAD

**Discussion by neilbmclaughlin** [PostgREST/postgrest#1772] [Q&A] [answered]
Just exploring postgrest and very impressed with it.

I like the name mapping ability as part of the get select syntax (e.g. select=contactId:contact_id). It means that an implementation detail (postgres column naming) doesn't leak into my code as I define my get url in configuration.

For posts the same isn't true (the post payload needs to use the column names) as I don't believe there is equivalent functionality for the columns parameter (e.g. columns=contactId:contact_id).

Is this something which has been considered?

Thanks in advance.

**Comment by wolfgangwalther** [comment, marked as answer]
I'm not sure whether this has been asked for or considered before - but I like it. Could you create an issue for that feature request? It will be hard to track in a discussion format - and I didn't find any button, yet, to move a discussion to an issue (it's possible the other way around).

**Comment by neilbmclaughlin** [comment]
Done, thanks for the quick response.
